pkgs/development/libraries/fmt/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl, cmake }:
+
+let
+  version = "5.2.1";
```

Comment:
You can put this variable in the derivation itself.

Reply:
Changed